src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+
+const populationList = [...populationElements].map((element) => {
```

Comment:
The same here. You can use directly `populationElements.map` because the `map` method returns a new array (this method doesn't mutate an original array)